pkg/chain/chain.go

```diff
@@ -152,7 +152,7 @@ type BondedECDSAKeep interface {
 	// All implementations should returns those events sorted by the
 	// block number in the ascending order.
 	PastSignatureSubmittedEvents(
-		keepAddress string,
+		keepAddress common.Address,
```

Comment:
We should stick with `string` if possible. It is a generic `chain` interface and we shouldn't use types from `go-ethereum`. It is true other functions are doing that but that's wrong and let's not escalate it.

Reply:
Changed, as you mentioned, wanted to stay consisted with other functions and didn't realize we went against the generic interface.